Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesManager.swift

```diff
@@ -388,6 +386,15 @@ public class ContentBlockerRulesManager: CompiledRuleListsSource {
                                                                                                 unprotectedSitesHash: nil))
             }
 
+            // todo - this hard code is a placeholder till I figure out how to check for this task vs. Ad attr task
+            if task.rulesList.name == "TrackerDataSet" {
+                if let perfInfo = result.performanceInfo {
+                    self.errorReporting?.fire(.contentBlockingCompilationTaskPerformance(retryCount: perfInfo.iterationCount,
```

Comment:
We want to exclude lookups right?

Reply:
`compilationCompleted` is only called when compilation happens, not when we do lookups. I also verified this with breakpoints and logs. Is there anything I’m missing?